doc/user/content/integrations/client-libraries/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "SQL drivers"
```

Comment:
Should I just rename the folder to be "SQL-drivers"? or rename the title of this page "Client libraries"?

What's currently being published uses client-libraries in the path but named it SQL drivers ... which is why I created an actual folder called client-libraries ... but it then seems strange to name it SQL drivers.